code/game/machinery/newscaster.dm

```diff
@@ -372,6 +380,14 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 			if(21)
 				dat+="<FONT COLOR='maroon'>Unable to print newspaper. Insufficient paper. Please notify maintenance personnel to refill machine storage.</FONT><BR><BR>"
 				dat+="<A href='?src=\ref[src];setScreen=[0]'>Return</A>"
+			if(22) //
+				dat+="<B>Comments:</B></BR>"
+				if(isemptylist(src.viewing_message.comments))
+					dat+="No comments on this story yet!</BR>"
+				else
+					for(var/datum/feed_comment/COMMENT in src.viewing_message.comments)
+						dat+="<BLOCKQUOTE style=\"padding:4px;border-left:4px #797979 solid;\"><B>\[[world.time]\] [COMMENT.author]:</B>[COMMENT.message]<BR></BLOCKQUOTE>"
+				dat+="<A href='?src=\ref[src];setScreen=[9]'>Return</A>"
 			else
 				dat+="I'm sorry to break your immersion. This shit's bugged. Report this bug to Agouri, polyxenitopalidou@gmail.com"
```

Comment:
Can you edit that line to "Report this bug on github".
And possibly search for other occurrences and change them aswell.